src/Sylius/Bundle/ApiBundle/Provider/CompositePaymentConfigurationProvider.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Provider;
+
+use Sylius\Bundle\ApiBundle\Payment\ApiPaymentMethodHandlerInterface;
+use Sylius\Component\Core\Model\PaymentInterface;
+use Sylius\Component\Core\Model\PaymentMethodInterface;
+
+/** @experimental */
+final class CompositePaymentConfigurationProvider implements PaymentConfigurationProviderInterface
```

Comment:
```suggestion
final class CompositePaymentConfigurationProvider implements CompositPaymentConfigurationProviderInterface
```

The current interface seems to be a better fit here: https://github.com/Sylius/Sylius/pull/12107/files#diff-891e0e19d6fdd944d01eca4ac4e136f9c24d86cb111a0e4a655d8a2465b53e59R20. It could stay as it is if we would drop support method (mentioned in previous comment)